public/cities.js

```diff
@@ -1,160 +1,149 @@
-
-    $('#add-city-button').click(function() {
-      $('#add-city-form').removeClass('invisible').addClass('visible');
-    });
-
-    $('#cancel-button').click(function() {
-      $('#add-city-form').removeClass('visible').addClass('invisible');
-      $('#error-message').removeClass('visible').addClass('invisible');
-      $('#name').val('');
-    });
-
-    $(document).ready(function() {
-      $('.closeModal').on('click', function(e) {
-        $('#interestModal').addClass('invisible');
-        $('#error-message-edit').removeClass('visible').addClass('invisible');
-      });
-      $.ajaxSetup({
-        headers: {
-          'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
-        }
-      });
-      $.ajax({
-        type: 'GET',
-        url: `api/cities${location.search}`,
-        cache: false,
-        processData: false,
-        contentType: false,
-        success: (cities) => {
-          $(".dynamic-tbody").html(generateTableRows(cities.data));
-        },
-        error: function(data) {
-          console.log(data);
-        }
-      });
-      $(document).on('click', 'button.delete', function() {
-        let id = $(this).data('id');
+$('#add-city-button').click(function() {
+  $('#add-city-form').removeClass('invisible').addClass('visible');
```

Comment:
I see a lot of adding and removing these two classes, and I think a function would make things clear.
Something like `hide(element)` and `show(element)` to replace those